### DISCUSSION TITLE
Wio-E5 Development Kit | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#215] [Q&A] [answered]
# Wio-E5 Development Kit | Seeed Studio Wiki

-->

https://wiki.seeedstudio.com/LoRa_E5_Dev_Board/

**Comment by scarbajali** [comment, marked as answer]
It took me awhile to figure out that it is required to install the USB driver for CP201x from [here](https://www.silabs.com/developers/usb-to-uart-bridge-vcp-drivers). Suggest to add this info to the documentation.

**Reply by MatthewJeffson** [reply]
Wow! Thank you for pointing it out!

**Comment by ronsnyder** [comment]
You also need to know that if you are using the ST LINK/V2 (20 pin version) you will need connect the following pins V2-Pin-1 to E5-SWO-VCC, V2-Pin-7 to E5-SWO-DIO, V1-Pin-9 to E5-SWO-CLK, V2-Pin-17 to E5-SWO-RST, V2-Pin-8 to E5-SWO-GND. As example, there are other pins you can use for VDD and GND.

**Comment by surumadurum** [comment]
There is no possibility to check battery state, correct? I could not find anything in the schematics like a voltage divider or a connection to an ADC pin...

**Reply by xzxcessarr** [reply]
Hello, well, you can use a board with the meter module and connect the battery and the E5 board, it can meter it and send the battery information to the E5

**Comment by surumadurum** [comment]
Hi, the wiki of the Lora E5 Dev Board states that external oscillators are used (High Speed External HSE and Low Speed External LSE), but after I ran into issues, I switched to Low Speed Internal, which resolved the problems.
After checking schematics, I could not find any external oscillators.
Are they just not in the schematic (but on the board) or is this a bug?

**Comment by ayhan01** [comment]
How do I connect to the helium network with Wio-E5 Development Kit? Is there an example that can explain step by step? For example, Wio-E5 Development is an application that shows when the button on the board is pressed. Or an application that will display the temperature read from the temperature sensor on the Wio-E5 Development board on the helium console.
Where I am helium 868 is covered in Europe. I made all the settings but I could not connect to the helium network. I tried both platforms below and failed.
https://console.iot-wireless.com/
https://console.helium-iot.xyz/front/

I did all the steps below.
https://iot-wireless.com/getting-started-chirpstack/

I learned Device EUI with the AT+ID command from the serial port in Arduino.
I set the key I generated in the Join EUI section on the console to wio-e5 with the “AT+KEY=APPKEY, “your-key” command.
I think I'm making the mistake in the OTAA KEY section. What exactly is OTAA KEY? What is the difference between the key where it says Join EU?

**Comment by steves2j** [comment]
FYI
For those trying to get I2C working and to get the 3v3 and 5v lines working, I2C is on chip2 with SDA on PA15 and SCL on PB15. So if you are using arduino Wire, you need to Wire.setSDA(PA15); Wire.setSCL(PB15); 
The Power lines are also controlled behind mosfets so assert (Make High) PA9(3v3) and PB10(5v)

**Comment by Sandeep5194** [comment]
while using at commands in serial monitor if i type AT in the serial monitor in response it is not showing ok and also only tx light is blinking in the wio e5 development board but rx light is not blinking 
can u suggest me how to counter this
should i erase firmware and reinstall it again

**Reply by nfs0619** [reply]
Thank you for your feedback. We have received your message and are currently arranging for debugging. If you do not receive further updates, please feel free to reach out to our technical support at [sensecap@seeed.cc](mailto:sensecap@seeed.cc) for further assistance. Thank you for your support and understanding.

**Comment by LoopOverlord1453** [comment]
Dear Seeed Studio Technical Team,

We have purchased two Wio-E5 Development Kits from Digi-Key with the part number 1597-113990934-ND, and we are experiencing range issues in P2P communication. When using the default LoRa settings with the AT firmware that comes preloaded on the devices, we observe good signal levels with RSSI: -2 dBm and SNR: 14. However, when I flash both LoRa devices with the FreeRTOS_LoRaWAN_AT project (Link: Seeed-Studio/LoRaWan-E5-Node at qian provided on your GitHub and linked on the Wio-E5 Dev Kit Wiki page) using STM32CubeProgrammer and run the firmware, the signal quality drops significantly — RSSI starts at around -70 dBm and SNR: 5. I’ve tested all the source code examples from your GitHub in STM32CubeIDE, but unfortunately, the results remain the same. At these signal levels, the communication link drops at about 50 meters.Additionally, when testing with Arduino using the RadioLib library (as discussed in this Seeed Studio Forum post), I again achieve strong signal readings of RSSI: -2 dBm and SNR: 14. Could you please share a working STM32CubeIDE example project for the Wio-E5 Development Kit where the RSSI and SNR values are at normal levels during P2P communication?

Best regards,
